### PR TITLE
http server feature

### PR DESCRIPTION
**Pull request by tpeacock19**
I've made an attempt at addressing #2, using an HTTP server rather than
the commandline. It has worked for me and I figured I would share in
case it would be useful for others.

### CODE REVIEW

**Comment by jcs090218** [comment]
Awesome! 👍 

Can you update our README file as well? And it seems like there is a compile error, see the following:

```
In toplevel form:
flymake-languagetool.el:165:1:Error: Unused lexical variable ‘objects’
make: *** [Makefile:19: unix-compile] Error 1
```

File: flymake-languagetool.el [hunk 1]
```diff
@@ -65,163 +87,168 @@
...
-(defun flymake-languagetool--start-timer ()
-  "Start the timer for grammar check."
-  (setq flymake-languagetool--source-buffer (current-buffer))
-  (when (timerp flymake-languagetool--timer) (cancel-timer flymake-languagetool--timer))
-  (setq flymake-languagetool--timer
-        (run-with-idle-timer flymake-languagetool-check-time nil
-                             #'flymake-languagetool--send-process)))
+(defun flymake-languagetool--output-to-errors (output source-buffer)
+  "Parse the JSON data from OUTPUT of LanguageTool. "
+  (let* ((objects nil)
```
Comment (jcs090218) [inline flymake-languagetool.el:167]: I think the variable is no longer needed?

Comment (tpeacock19) [reply flymake-languagetool.el]: You are right. I've removed this.

**Comment by tpeacock19** [comment]
I've updated to README to include an example using LanguageTool's remote api as well. Let me know if it makes sense or I need to update anything else further.

**Comment by jcs090218** [comment]
You might want to forward declare the variable `url-http-end-of-headers` to resolve the latest CI.

```
In toplevel form:
flymake-languagetool.el:181:14:Error: reference to free variable ‘url-http-end-of-headers’
make: *** [Makefile:19: unix-compile] Error 1
```

**Comment by tpeacock19** [comment]
I wasn't actually sure how to address that error. I think i've got it now though.

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
Thanks! :)

**Comment by jcs090218** [comment]
I think you have made decent amount of changes. Wonder if you would like to join the org and take some control of this package (to co-maintain)? Furthermore, feel free to open another to add yourself as an author if you like to (top of the file).

@tpeacock19 WDYT?

**Comment by tpeacock19** [comment]
Yeah I would be happy to join. It'd be my first org so I'm not familiar with the process, but just let me know what I need to do.

**Comment by jcs090218** [comment]
Great! Just sent you the invitation!

No worries! That's why it's great to have people co-maintain an open source project. :)

**Pull request merged**